process.go

```diff
@@ -21,7 +17,4 @@ type ProcessConfig struct {
 	Stdin  io.Reader
 	Stdout io.Writer
 	Stderr io.Writer
-
-	// Console is the path to the pty slave for use by the master
-	Console string
```

Comment:
The general idea is that the container configuration specifies **how** a container and it's processes execute.  The console, env, capabilities, rlimits, etcd.  Then the process specifies **what** is run.  Args.  I guess we can make the argument that env should be removed from the container and put at the process level as well as user( maybe ).